.github/workflows/implement-ready-issues.yml

```
@@ -0,0 +1,160 @@
+        env:
+          GH_TOKEN: ${{ secrets.COMMIT_GITHUB_TOKEN || secrets.GITHUB_TOKEN }}
+        run: |
+          BRANCH="${{ steps.plan.outputs.branch }}"
+          ISSUE_NUMBER="${{ steps.plan.outputs.issue_number }}"
+          git add -A
+          if git diff --cached --quiet; then
+            echo "No implementation changes beyond the plan commit."
+          else
+            git commit -m "feat: implement issue #${ISSUE_NUMBER}
+
+Closes #${ISSUE_NUMBER}
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)